modules/Individual Needs/investigations_manage_add.php

```diff
@@ -0,0 +1,115 @@
+
+if (isActionAccessible($guid, $connection2, '/modules/Individual Needs/investigations_manage_add.php') == false) {
+    //Acess denied
+    echo "<div class='error'>";
+    echo __('You do not have access to this action.');
+    echo '</div>';
+} else {
+    //Get action with highest precendence
+    $highestAction = getHighestGroupedAction($guid, $_GET['q'], $connection2);
+    if ($highestAction == false) {
+        echo "<div class='error'>";
+        echo __('The highest grouped action cannot be determined.');
+        echo '</div>';
+    } else {
+        $page->breadcrumbs
+            ->add(__('Manage Investigations'), 'investigations_manage.php')
+            ->add(__('Add'));
+
+        $gibbonPersonID = $_GET['gibbonPersonID'] ?? '';
+        $gibbonRollGroupID = $_GET['gibbonRollGroupID'] ?? '';
+        $gibbonYearGroupID = $_GET['gibbonYearGroupID'] ?? '';
+
+        $editLink = '';
+        $editID = '';
+        if (isset($_GET['editID'])) {
+            $editID = $_GET['editID'];
+            $editLink = $_SESSION[$guid]['absoluteURL']."/index.php?q=/modules/Individual Needs/investigations_manage_edit.php&gibbonINInvestigationID=$editID&gibbonPersonID=$gibbonPersonID&gibbonRollGroupID=$gibbonRollGroupID&gibbonYearGroupID=$gibbonYearGroupID";
+        }
+        if (isset($_GET['return'])) {
+            returnProcess($guid, $_GET['return'], $editLink);
+        }
+
+        if ($gibbonPersonID != '' or $gibbonRollGroupID != '' or $gibbonYearGroupID != '') {
+            echo "<div class='linkTop'>";
+            echo "<a href='".$_SESSION[$guid]['absoluteURL']."/index.php?q=/modules/Individual Needs/investigations_manage.php&gibbonPersonID=$gibbonPersonID&gibbonRollGroupID=$gibbonRollGroupID&gibbonYearGroupID=$gibbonYearGroupID'>".__('Back to Search Results').'</a>';
+            echo '</div>';
+        }
+
+        $form = Form::create('addform', $_SESSION[$guid]['absoluteURL']."/modules/Individual Needs/investigations_manage_addProcess.php?gibbonPersonID=$gibbonPersonID&gibbonRollGroupID=$gibbonRollGroupID&gibbonYearGroupID=$gibbonYearGroupID");
+        $form->setFactory(DatabaseFormFactory::create($pdo));
+        $form->addHiddenValue('address', "/modules/Individual Needs/investigations_manage_add.php");
+        $form->addRow()->addHeading(__('Basic Information'));
+        
+        //Student
+        $row = $form->addRow();
+        	$row->addLabel('gibbonPersonIDStudent', __('Student'));
+        	$row->addSelectStudent('gibbonPersonIDStudent', $_SESSION[$guid]['gibbonSchoolYearID'])->placeholder(__('Please select...'))->selected($gibbonPersonID)->required();
+
+        //Status
+        $row = $form->addRow();
+        	$row->addLabel('status', __('Status'));
+        	$row->addTextField('status')->setValue(__('Referral'))->required()->readonly();
+
+        //Date
+        $row = $form->addRow();
+        	$row->addLabel('date', __('Date'))->description($_SESSION[$guid]['i18n']['dateFormat'])->prepend(__('Format:'));
```

Comment:
Description can be removed, dates automatically add the format info.

```suggestion
        	$row->addLabel('date', __('Date'));
```